facebookads/api.py

```diff
@@ -489,25 +513,40 @@ def execute(self):
         retry_indices = []
 
         for index, response in enumerate(responses):
-            if response:
-                body = response.get('body')
-                code = response.get('code')
-                headers = response.get('headers')
-
+            if not response:
+                transient_body = {
+                    "error": {
+                        "is_transient": True,
+                        "message": "Transient Error",
+                        "type": "empty response"
+                    }
+                }
                 inner_fb_response = FacebookResponse(
```

Comment:
I figured it was better to call the callbacks with a fake transient response when it's empty rather than nothing, for consistency purposes